.github/workflows/build.yml

```diff
@@ -21,12 +21,11 @@ jobs:
         path: ${{ github.event_path }}
 
   build-test:
-    # Python version < 3.7 requires ubuntu-20.04
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.6", "3.x"]
+        python-version: ["3.9", "3.x"]
```

Comment:
10+ percent of users are still on older versions so it's probably better to still have some test coverage. 

https://pypistats.org/packages/junitparser